dashboard/grid_builder.py

```diff
@@ -101,12 +109,19 @@ def get_result(self,
                             May be None.>
             }
         """
+        self_result = []
+        pair_result = []
+        cell_color_status = 'Self-SUCCESS'
```

Comment:
does it make sense to always use the lowercase name to make the names easier to grep for, etc.?

Reply:
Done.